.NET/Microsoft.Recognizers.Definitions.Common/Microsoft.Recognizers.Definitions.Common.csproj

```diff
@@ -440,6 +461,10 @@
     <DotNetCliToolReference Include="T5.TextTransform.Tool" Version="1.1.0-*" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Please be careful not to add spurious changes to the project files. What is this service reference?

Reply:
removed